README.adoc

```diff
@@ -0,0 +1,208 @@
+image:https://goreportcard.com/badge/github.com/aerogear/mobile-security-service[Go Report Card (Go Report Card), link=https://goreportcard.com/report/github.com/aerogear/mobile-security-service]
+endif::[]
+
+
+
+== Overview
+
+This is the server component of the AeroGear Mobile Security Service. It is a RESTful API that allows developers to view, enable and disable specific versions of applications on demand, with the information stored in a PostgreSQL database.
+
+== Prerequisites
+
+.Prerequisites
+|===
+|https://golang.org/doc/install[Install Golang]
+|https://github.com/golang/go/wiki/SettingGOPATH[Ensure the $GOPATH environment variable is set]
+|https://docs.docker.com/compose/install/[Install the dep package manager]
+|https://docs.docker.com/compose/install/[Install Docker and Docker Compose]
+|===
+
+See the https://aerogear.org/community/#guides[Contributing Guide] for more information.
+
+== Getting Started
+
+If you'd like to simply run the entire application in `docker-compose`, follow [these instructions](#Run-Entire-Application-with-Docker-Compose).
+
+Golang projects are kept in a [workspace](https://golang.org/doc/code.html#Workspaces) that follows a very specific architecture. Before cloning this repo, be sure you have a `$GOPATH` environment variable set up.
+
+:numbered:
+=== Clone the Repository
+
+[source,shell]
+----
+git clone git@github.com:aerogear/mobile-security-service.git $GOPATH/src/github.com/aerogear/mobile-security-service
+----
+
+=== Install Dependencies
+
+[source,shell]
+----
+make setup
+----
+
+NOTE: This is using the `dep` package manager under the hood. You will see the dependencies installed in the `vendor` folder.
+
+=== Start the Server
+
+[source,shell]
+----
+go run cmd/mobile-security-service/main.go
+----
+
+=== Run Entire Application with Docker Compose
+
+This section shows how to start the entire application with `docker-compose`. This is useful for doing some quick tests (using the SDKs) for example.
+
+First, compile a Linux compatible binary:
+
+[source,shell]
+----
+go build -o mobile-security-service cmd/mobile-security-service/main.go
+----
+
+This binary will be used to build the Docker image. Now start the entire application.
+
+[source,shell]
+----
+docker-compose up
+----
+:numbered!:
+
+== Setup and configurations
+
+=== Environment Variables
+
+The **mobile-security-service** is configured using environment variables.
+
+* By default, the application will look for system environment variables to use.
+* If a system environment variable cannot be found, the application will then check the `.env` file in the application root.
+* If the `.env` file does not exist, or if the variable is not defined in the file, the application will use the default value defined in [config.go](./pkg/config/config.go).
```

Comment:
this hyperlink is broken.

Reply:
👍 fixed

Reply:
Have you pushed @camilamacedo86 ? This change does not show up yet.

Reply:
Could you check the latest version please?

Reply:
https://github.com/aerogear/mobile-security-service/blob/dca2fda16b78616c405db583d759d7b22ec91ad2/README.adoc

Reply:
![screenshot from 2019-02-08 16-20-43](https://user-images.githubusercontent.com/11743717/52490954-dbebfc80-2bbd-11e9-9428-b6c57ab58f1d.png)

It is still the same. See from your own branch:

https://github.com/camilamacedo86/mobile-security-service/tree/AEROGEAR-8506#environment-variables

Reply:
You are correct. I found the link and the issue. Solved now.

<img width="948" alt="screenshot 2019-02-08 at 16 29 29" src="https://user-images.githubusercontent.com/7708031/52491378-e2c73f00-2bbe-11e9-81a2-9f6492693b99.png">